
# print('Olá, Mundo!')
# SyntaxError: Non-UTF-8 code starting with '\xe1' in file C:/my file/03.python/????/fluent_python/???????????/py4-8?ola.py?“??????”??????.py on line 2, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
# z??# coding: cp1252
# print('Olá, Mundo!')



